apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/logs/_components/DocumentLogs/index.tsx

```diff
@@ -22,7 +22,7 @@ import { DocumentLogsTable } from './DocumentLogsTable'
 
 const useDocumentLogSocket = (
   documentUuid: string,
-  mutate: ReturnType<typeof useDocumentLogs>['mutate'],
+  mutate: ReturnType<typeof useDocumentLogs<false>>['mutate'],
```

Comment:
Honestly, this worked better than I expected. This `<false>` is synchronized with a query string. 

I don't even know how this works: `?expectedErrors=true` is honored and if you set it to false this store returns logs with errors (typed). If not plain logs